### PR TITLE
When '--builder' isn't specified, use the WLSIMG_BUILDER env var...

### PR DESCRIPTION
**Pull request by tbarnes-us**
…as the build engine (if set). Continue to use 'docker' otherwise.

### CODE REVIEW

**Review by ddsharpe — APPROVED** [review]

File: documentation/1.11/content/userguide/tools/create-aux-image.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Usage: imagetool createAuxImage [OPTIONS]
 | `--tag` | (Required) Tag for the final build image. Example: `store/oracle/mydomain:1`  |   |
 | `--additionalBuildCommands` | Path to a file with additional build commands. For more details, see [Additional information](#--additionalbuildcommands). |
 | `--additionalBuildFiles` | Additional files that are required by your `additionalBuildCommands`.  A comma separated list of files that should be copied to the build context. See [Additional information](#--additionalbuildfiles). |
-| `--builder`, `-b` | Executable to process the Dockerfile. Use the full path of the executable if not on your path. | `docker`  |
+| `--builder`, `-b` | Executable to process the Dockerfile. Use the full path of the executable if not on your path. | Defaults to 'docker', or, when set, to the value in environment variable WLSIMG_BUILDER. |
```
Comment (rosemarymarano) [inline documentation/1.11/content/userguide/tools/create-aux-image.md:27]: WLSIMG_BUILDER -> `WLSIMG_BUILDER` (env variable should be in code font; please correct globally)

File: documentation/1.11/content/userguide/tools/inspect-image.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ Usage: imagetool inspect [OPTIONS]
 | Parameter | Definition | Default |
 | --- | --- | --- |
 | `--image`, `-i` | (Required) The image ID or image name to be inspected.  |   |
-| `--builder`, `-b` | Executable to process the Dockerfile. Use the full path of the executable if not on your path. | `docker`  |
+| `--builder`, `-b` | Executable to inspect docker images. Use the full path of the executable if not on your path. | Defaults to 'docker', or, when set, to the value in environment variable WLSIMG_BUILDER. |
```
Comment (rosemarymarano) [inline documentation/1.11/content/userguide/tools/inspect-image.md:19]: docker  -> Docker

**Review by rosemarymarano — COMMENTED** [review]
Minor edits

File: documentation/1.11/content/userguide/tools/create-aux-image.md (hunk 1 above)
Comment (tbarnes-us) [reply documentation/1.11/content/userguide/tools/create-aux-image.md]: Done. Also changed 'docker' to `docker`.

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**